common/utils/__init__.py

```diff
@@ -0,0 +1,7 @@
+from os import environ
+
+def get_env_var(key):
```

Comment:
I would call this something like `safe_get_env_var` so we know this function will also be in charge of checking if the env var is present and raising and error if not 🙂